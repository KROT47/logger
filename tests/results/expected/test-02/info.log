[2020-03-12T22:07:43.644Z] INFO on 02 | { x: 1, _circular: true, a: [Circular] }
[2020-03-12T22:07:43.645Z] INFO on 02 | { a: { a: { a: { a: { a: { a: '[Object]' } } } } } }
[2020-03-12T22:07:43.646Z] INFO on 02 | { a: { a: { a: { a: { a: { a: '[Array]' } } } } } }
[2020-03-12T22:07:43.646Z] INFO on 02 | { a: { a: { a: { a: { a: { a: [Function: a] } } } } } }
[2020-03-12T22:07:43.647Z] INFO on 02 | { a: { a: { a: { a: { a: { a: NaN } } } } } }
[2020-03-12T22:07:43.647Z] INFO on 02 | { a: { a: { a: { a: { a: { a: undefined } } } } } }
[2020-03-12T22:07:43.647Z] INFO on 02 | { a: { a: { a: { a: { a: { a: null } } } } } }
[2020-03-12T22:07:43.648Z] INFO on 02 | { a: { a: { a: { a: { a: { a: 1 } } } } } }
[2020-03-12T22:07:43.648Z] INFO on 02 | { a: { a: { a: { a: { a: { a: '1' } } } } } }
[2020-03-12T22:07:43.649Z] INFO on 02 | { a: { a: { a: { a: { a: { a: 1970-01-01T00:00:00.000Z } } } } } }
[2020-03-12T22:07:43.650Z] INFO on 02 | { a: { a: { a: { a: { a: { a: [Error: AAA] } } } } } }
[2020-03-12T22:07:43.651Z] INFO on 02 | [ 1, [ 1, [ 1, [ 1, [ 1, [ 1, '[Array]' ] ] ] ] ] ]
[2020-03-12T22:07:43.651Z] INFO on 02 | { a: { a: { a: { a: { a: /a/ } } } } }
[2020-03-12T22:07:43.652Z] INFO on 02 | { a: { a: { a: { a: { a: Symbol(asd) } } } } }
[2020-03-12T22:07:43.653Z] INFO on 02 | { a: { a: { a: { a: { a: Promise { 'asd' } } } } } }
