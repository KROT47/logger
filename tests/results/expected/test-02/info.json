{"ts":"2020-03-12T22:07:43.643Z","level":"info","hostname":"02","msgs":[{"x":1,"_circular":"[true]","a":{"x":1,"_circular":"[true]","a":{"x":1,"_circular":"[true]","a":{"x":1,"_circular":"[true]","a":{"x":1,"_circular":"[true]","a":{"x":1,"_circular":"[true]","a":"[Object]"}}}}}}]}
{"ts":"2020-03-12T22:07:43.645Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Object]"}}}}}}]}
{"ts":"2020-03-12T22:07:43.646Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Array]"}}}}}}]}
{"ts":"2020-03-12T22:07:43.646Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Function]"}}}}}}]}
{"ts":"2020-03-12T22:07:43.646Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[NaN]"}}}}}}]}
{"ts":"2020-03-12T22:07:43.647Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"[Undefined]"}}}}}}]}
{"ts":"2020-03-12T22:07:43.647Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":null}}}}}}]}
{"ts":"2020-03-12T22:07:43.648Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":1}}}}}}]}
{"ts":"2020-03-12T22:07:43.648Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"1"}}}}}}]}
{"ts":"2020-03-12T22:07:43.649Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":"1970-01-01T00:00:00.000Z"}}}}}}]}
{"ts":"2020-03-12T22:07:43.649Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":{"a":{"message":"AAA","stack":"[Error: AAA]","_type":"Error"}}}}}}}]}
{"ts":"2020-03-12T22:07:43.651Z","level":"info","hostname":"02","msgs":[[1,[1,[1,[1,[1,[1,"[Array]"]]]]]]]}
{"ts":"2020-03-12T22:07:43.651Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":"[/a/]"}}}}}]}
{"ts":"2020-03-12T22:07:43.652Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":"[Symbol(asd)]"}}}}}]}
{"ts":"2020-03-12T22:07:43.653Z","level":"info","hostname":"02","msgs":[{"a":{"a":{"a":{"a":{"a":"[Promise { 'asd' }]"}}}}}]}
